src/items/generics.md

```diff
@@ -227,6 +227,22 @@ fn generic<const B: bool>() {
 }
 ```
 
+r[items.generics.const.inferred]
+
+r[items.generics.const.inferred.syntax]
+```grammar,types
+InferredConst -> 
+      `_`
+    | (InferredConst)
```

Comment:
```suggestion
    | `(` InferredConst `)`
```

Reply:
okay @ root didn't fix anything. I'm not sure how i'm supposed to use this anywhere